src/common/utils.ts

```diff
@@ -925,6 +925,21 @@ export const samplesFullQueryDescriptionFields =
 }\n \
   </pre>';
 
+export const filterUserIdentityExample = '{ "where": { "field": "value" }}';
+
+export const filterUserIdentityDescription =
+  '<pre>\n \
+{\n \
+  "field": "value"\n \
+}\n \
+  or \n \
+{\n \
+  "where?": {\n \
+    "field": "value"\n \
+  }\n \
+}\
+</pre>';
+
```

Comment:
It's useful to allow both a key-value pair like "profile.email":"value" and just "value" without a key to work interchangeably. 

Since this endpoint is meant for validating emails, using "profile.email":"value" instead of "field":"value" would be clearer for users. 

Additionally, the "where?":{"field":"value"} format is currently only used by the frontend, and should be deprecated for this endpoint in the future, as it can be confusing.